AppCenterDataStorage/AppCenterDataStorage/Internal/Client/MSTokenExchange.m

```diff
@@ -34,6 +36,12 @@ + (void)performDbTokenAsyncOperationWithHttpClient:(MSStorageIngestion *)httpCli
     NSData *payloadData = [NSJSONSerialization dataWithJSONObject:@{kMSPartitions : @[ partition ]} options:0 error:&jsonError];
 
     // Http call.
```

Comment:
This comment seems unnecessary to me. The variable before it is called `payloadData` and the logic below is like 10 lines of self-explanatory code.